cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -106,7 +107,7 @@ def test_cluster_section_from_250_cfn(mocker, cfn_params_dict, expected_section_
 @pytest.mark.parametrize(
     "cfn_params_dict, expected_section_dict",
     [
-        ({}, DefaultDict["cluster"].value),
+        ({}, utils.merge_dicts(DefaultDict["cluster"].value, {"additional_iam_policies": []})),
```

Comment:
We could add another test for 250_cfn with batch as a scheduler to have a test with an additional_iam_policies with the AWSBatchPolicy inside.

Reply:
From Cfn we load directly only existing policies